plugins/SplitMerge/class.splitmerge.plugin.php

```diff
@@ -218,11 +218,10 @@ public function moderationController_mergeDiscussions_create($Sender) {
                     $CommentModel->Validation->results(true);
                     $CommentID = $CommentModel->save($Comment);
                     if ($CommentID) {
-                        // Move any attachments (FileUpload plugin awareness)
-                        if (class_exists('MediaModel')) {
-                            $MediaModel = new MediaModel();
-                            $MediaModel->reassign($Discussion['DiscussionID'], 'discussion', $CommentID, 'comment');
-                        }
+                        $Comment['CommentID'] = $CommentID;
+                        $this->EventArguments['SourceDiscussion'] = $Discussion;
+                        $this->EventArguments['TargetComment'] = $Comment;
+                        $this->fireEvent('TransformDiscussionToComment');
```

Comment:
This would make more sense to `fireAs('ModerationController')`, I think. Hooking into a plugin-fired event otherwise requires the use of `base_` everywhere.